qiskit/compiler/transpile.py

```diff
@@ -308,13 +355,21 @@ def _transpile_circuit(circuit_config_tuple: Tuple[QuantumCircuit, Dict]) -> Qua
     if ms_basis_swap is not None:
         pass_manager.append(MSBasisDecomposer(ms_basis_swap))
 
+    if pass_manager_config.scheduling_method is not None:
```

Comment:
Why is this added here and not in level_x_pass_manager? I'm not that familiar with the transpiler layout yet

Reply:
Because I don't think those depend on `optimization_level` (even in the future), while I put scheduling passes in level_x_pass_manager since they can be varied depending on `optimization_level` in the future (that's why we introduce `scheduling_method`). Of cause we can add them in level_x_pass_manager but I just avoided unnecessary copy and paste.